kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/KFuzzPlugin.kt

```diff
@@ -48,7 +54,11 @@ abstract class FuzzTask : Test() {
 
 @Suppress("unused")
 fun Project.fuzzConfig(block: KFuzzConfigBuilder.() -> Unit) {
-    val config = KFuzzConfigBuilder.build(block)
+    val buildDir = layout.buildDirectory.get()
+    val config = KFuzzConfigBuilder.build {
+        workDir = buildDir.dir("fuzz").asFile.toPath()
```

Comment:
You are letting the user specify `workDir` and then overriding it (You can test this behavior by adding workDir in the `kotlinx.fuzz.test/build.gradle.kts`). You shouldn't let the user do it or you shouldn't change it if the user has provided something

Reply:
Yeah, agree. I will make a quick hack to fix this and then think on the proper solution

Reply:
Ok, I allowed to set properties multiple times in `build` block instead